ample/util/benchmark_util.py

```diff
@@ -309,7 +309,7 @@ def analyseModels(amoptd):
             structure_list = [amoptd['native_pdb_std']]
             amoptd['tmComp'] = tm.compare_structures(model_list, structure_list, fastas=[amoptd['fasta']])
         except Exception as e:
-            logger.critical("Unable to run TMscores: %s", e)
+            logger.exception("Unable to run TMscores: %s", e)
```

Comment:
you don't need to include the exception. `logger.exception` prints the traceback automatically. you're call would look something like
```python
logger.exception("Unable to run TMscores") 
```

Reply:
```python
>>> import logging
>>> logging.basicConfig()
>>> try:
...     raise
... except:
...     logging.exception('Example')
... 
ERROR:root:Example
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
RuntimeError: No active exception to reraise
```